person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def getting_age
+    @age
+  end
+
+  def getting_name
+    @name
+  end
+
+  def getting_id
+    @id
+  end
+
+  def setting_age
+    @age = 12
+  end
+
+  def setting_name
+    @name = 'Billions'
+  end
```

Comment:
- [ ] Your getters and setters should be set using the `attr_reader` (getter) and `attr_accessor` (getter and setter) shortcuts keywords. You should have getter for `id` and a getter-setter for `name` and `age`.

  Instead of all these declarations, kindly refactor to something like this:
  ```
  attr_accessor :name, :age 
  attr_reader :id
  ```